pages/faq/troubleshooting/troubleshooting.md

```diff
@@ -151,6 +152,10 @@ NOTE: On Jetson devices it is important to not interrupt the balenaOS update pro
 
 If the device is running, typing `uname -a` will show the L4T in the kernel version string. Additionally, the L4T version is visible in the [balena-jetson github repository][tegra-binaries-version] for every release tag.
 
+### Custom device tree support
+
+Loading of custom device trees (DT) in balenaOS is supported only by the Jetson Nano and Jetson TX2 family of devices, which have u-boot support. The complete list of devices supporting custom DT and how to apply them is available in the [hardware section](https://www.balena.io/docs/learn/develop/hardware/i2c-and-spi/#jetson-devices).
```

Comment:
We should use relative cross-link URLs to other docs sections if possible, and not the absolute URL with the host.

Reply:
Thanks Kyle, I updated the link